### DISCUSSION TITLE
Ability to group medications.

### DISCUSSION THREAD

**Discussion by caj411** [diegopvlk/Dosage#51] [Ideas]
I take most of my medications on a morning or nightly schedule, ie twice daily.  If I could group them as such, it would make it much easier for me to mark taken or missed.  I would like to be able to group my morning meds into a group and my nightly meds into a group, then I could check taken or missed for the entire group, only two clicks. Otherwise I'm having to check 20 medications as taken or missed twice daily. Too cumbersome for me to use that feature if selecting each drug individually.

**Comment by diegopvlk** [comment]
They're already grouped by time, wouldn't set all your morning medications to a time like 8:00 and night medications to 19:00 works?

I like the group selection idea, I'll see what I can do.

**Reply by caj411** [reply]
I have them set by time, but I get a pop-up message for each individual medicine.  What would be nice would be a single pop-up for "morning meds", mid-day meds", or "evening meds" or whatever grouping name I have input, so one click would acknowledge that I have taken all my meds for that group as opposed to 20 pop-ups with 20 clicks.

Most people like myself do not open 20 individual bottles daily, we do that once a week and put all the pills into a separator for M-F, then all I have to do is open my morning/mid-day/evening meds separator for that day and time and take them. I literally take a handful of pills at a time, its hard to keep track, and this app really helps.

![screenshot](https://github.com/diegopvlk/Dosage/assets/34217875/660c6fe5-a450-4727-bcce-a101cd65e02b)

**Comment by diegopvlk** [comment]
Next update will have have grouped notifications (I won't make it re-nameable though) and the ability to select the grouped medications. 

If you want to test it: [io.github.diegopvlk.Dosage.zip](https://github.com/diegopvlk/Dosage/files/14574353/io.github.diegopvlk.Dosage.zip)

Click the time to select the group.

**Reply by caj411** [reply]
I installed the version 1.5.6 from the link above, but I'm not seeing how to group medications.  I click the time, but not seeing any way to group them unless I'm in the wrong place.
![Screenshot from 2024-03-13 14-11-52](https://github.com/diegopvlk/Dosage/assets/34217875/ff20bf93-3097-48e7-b5bd-49ac04ab890d)

**Reply by diegopvlk** [reply]
You don't need to do anything

![image](https://github.com/diegopvlk/Dosage/assets/25941156/fb664cb7-7b93-48a9-9955-37d437a8d3e6)

**Reply by caj411** [reply]
Ah, so subtle I didn't even notice.  Good job, thanks.
